Chat: describe API (#6640)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxChat/1 Configuration/typingUsers.md

---
---
##### shortDescription
An array of users who are currently typing.

---

> The following image shows messages that appear when a single or multiple users are typing:

Reviewer: ```suggestion
The following image displays messages that appear when a single user or multiple users are typing:
```

> The following image shows the message types displayed when different numbers of users are typing:

Reviewer 2: ```suggestion
The following image shows messages that appear when a single or multiple users are typing:
```

> ![The message types displayed when different numbers of users are typing](/images/Chat/typing-users.png)

Reviewer 2: ```suggestion
![Messages that display the number of users typing](/images/Chat/typing-users.png)
```
